<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home_Page</class>
 <widget class="QWidget" name="Home_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>虚拟股票交易平台</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>760</width>
     <height>430</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #87CEFA;  /* 按钮背景颜色，淡蓝色 */
    border: 2px solid #1E90FF;  /* 边框颜色，深蓝色 */
    color: white;               /* 字体颜色 */
    padding: 10px 20px;         /* 内边距 */
    text-align: center;         /* 文本居中 */
    font-size: 16px;            /* 字体大小 */
    border-radius: 10px;        /* 边角圆润 */
    font-family: Arial, sans-serif;  /* 字体 */
}

QPushButton:hover {
    background-color: #00BFFF;  /* 鼠标悬停时的背景颜色 */
    border: 2px solid #1E90FF;  /* 悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #1E90FF;  /* 按下时的背景颜色 */
    border: 2px solid #87CEFA;  /* 按下时的边框颜色 */
}

QPushButton:disabled {
    background-color: #B0C4DE;  /* 禁用时的背景颜色 */
    color: #A9A9A9;             /* 禁用时的字体颜色 */
}
</string>
   </property>
   <property name="text">
    <string>股市资讯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mystock_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #87CEFA;  /* 按钮背景颜色，淡蓝色 */
    border: 2px solid #1E90FF;  /* 边框颜色，深蓝色 */
    color: white;               /* 字体颜色 */
    padding: 10px 20px;         /* 内边距 */
    text-align: center;         /* 文本居中 */
    font-size: 16px;            /* 字体大小 */
    border-radius: 10px;        /* 边角圆润 */
    font-family: Arial, sans-serif;  /* 字体 */
}

QPushButton:hover {
    background-color: #00BFFF;  /* 鼠标悬停时的背景颜色 */
    border: 2px solid #1E90FF;  /* 悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #1E90FF;  /* 按下时的背景颜色 */
    border: 2px solid #87CEFA;  /* 按下时的边框颜色 */
}

QPushButton:disabled {
    background-color: #B0C4DE;  /* 禁用时的背景颜色 */
    color: #A9A9A9;             /* 禁用时的字体颜色 */
}
</string>
   </property>
   <property name="text">
    <string>我的股市</string>
   </property>
  </widget>
  <widget class="QPushButton" name="community_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #87CEFA;  /* 按钮背景颜色，淡蓝色 */
    border: 2px solid #1E90FF;  /* 边框颜色，深蓝色 */
    color: white;               /* 字体颜色 */
    padding: 10px 20px;         /* 内边距 */
    text-align: center;         /* 文本居中 */
    font-size: 16px;            /* 字体大小 */
    border-radius: 10px;        /* 边角圆润 */
    font-family: Arial, sans-serif;  /* 字体 */
}

QPushButton:hover {
    background-color: #00BFFF;  /* 鼠标悬停时的背景颜色 */
    border: 2px solid #1E90FF;  /* 悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #1E90FF;  /* 按下时的背景颜色 */
    border: 2px solid #87CEFA;  /* 按下时的边框颜色 */
}

QPushButton:disabled {
    background-color: #B0C4DE;  /* 禁用时的背景颜色 */
    color: #A9A9A9;             /* 禁用时的字体颜色 */
}
</string>
   </property>
   <property name="text">
    <string>股市交流</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trade_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #87CEFA;  /* 按钮背景颜色，淡蓝色 */
    border: 2px solid #1E90FF;  /* 边框颜色，深蓝色 */
    color: white;               /* 字体颜色 */
    padding: 10px 20px;         /* 内边距 */
    text-align: center;         /* 文本居中 */
    font-size: 16px;            /* 字体大小 */
    border-radius: 10px;        /* 边角圆润 */
    font-family: Arial, sans-serif;  /* 字体 */
}

QPushButton:hover {
    background-color: #00BFFF;  /* 鼠标悬停时的背景颜色 */
    border: 2px solid #1E90FF;  /* 悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #1E90FF;  /* 按下时的背景颜色 */
    border: 2px solid #87CEFA;  /* 按下时的边框颜色 */
}

QPushButton:disabled {
    background-color: #B0C4DE;  /* 禁用时的背景颜色 */
    color: #A9A9A9;             /* 禁用时的字体颜色 */
}
</string>
   </property>
   <property name="text">
    <string>股市交易</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #87CEFA;  /* 按钮背景颜色，淡蓝色 */
    border: 2px solid #1E90FF;  /* 边框颜色，深蓝色 */
    color: white;               /* 字体颜色 */
    padding: 10px 20px;         /* 内边距 */
    text-align: center;         /* 文本居中 */
    font-size: 16px;            /* 字体大小 */
    border-radius: 10px;        /* 边角圆润 */
    font-family: Arial, sans-serif;  /* 字体 */
}

QPushButton:hover {
    background-color: #00BFFF;  /* 鼠标悬停时的背景颜色 */
    border: 2px solid #1E90FF;  /* 悬停时的边框颜色 */
}

QPushButton:pressed {
    background-color: #1E90FF;  /* 按下时的背景颜色 */
    border: 2px solid #87CEFA;  /* 按下时的边框颜色 */
}

QPushButton:disabled {
    background-color: #B0C4DE;  /* 禁用时的背景颜色 */
    color: #A9A9A9;             /* 禁用时的字体颜色 */
}
</string>
   </property>
   <property name="text">
    <string>管理员入口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
